example/30-worker.yaml

```diff
@@ -84,5 +84,13 @@ spec:
     volume:
       type: pd-standard
       size: 20Gi
+    # workerconfig is used to specify the interface of gcp local ssd
+    dataVolumes:
+      type: SCRATCH
+      #interface: NVME
+    data
```

Comment:
I don't understand this example, the `dataVolumes` has no field `interface`: https://github.com/gardener/gardener/blob/master/pkg/apis/extensions/v1alpha1/types_worker.go#L144-L156
Also, what is `data` inline 91?

Reply:
Actually, the interface is read from workerconfig. And i remove it